nbbuild/antsrc/org/netbeans/nbbuild/assets/package.html

```diff
@@ -0,0 +1,31 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+Tasks responsible for generating the following Apache NetBeans assets:
+- Apache NetBeans Platform splash image
+- Apache NetBeans IDE splash image
+- Apache NetBeans About image
+See <a href="https://issues.apache.org/jira/browse/NETBEANS-2455">NETBEANS-2455</a>.
+
+This directory contains also the following assets:
+
+- apache-netbeans-splash.svg - InkScape compatible SVG file.
+- apache-netbeans-splash.png - Template PNG file for screen image, with size 640x396
+- apache-netbeans-about.png - Template PNG file for about image, with size 529x327
```

Comment:
I wonder if it would make sense to have an example/command line on how the SVG should be converted to the PNG here?